config/development.json

```diff
@@ -57,6 +57,7 @@
 		"help": true,
 		"help/courses": true,
 		"jetpack/api-cache": true,
+		"jetpack/branded-connection": true,
```

Comment:
I'd probably prefer `jetpack/connection-rebranding` but I also trust your judgement on the best choice of words, so if you prefer yours more, just :shipit:

Reply:
👍 I don't care too much, went with yours 🙂